mypy/test/testparse.py

```diff
@@ -7,25 +7,29 @@
 from mypy import defaults
 from mypy.myunit import Suite, AssertionFailure
 from mypy.test.helpers import assert_string_arrays_equal
-from mypy.test.data import parse_test_cases, DataDrivenTestCase
+from mypy.test.data import parse_test_cases, DataDrivenTestCase, DataSuite
 from mypy.test import config
 from mypy.parse import parse
 from mypy.errors import CompileError
 from mypy.options import Options
 
 
-class ParserSuite(Suite):
+class ParserSuite(DataSuite):
     parse_files = ['parse.test',
                    'parse-python2.test']
 
-    def cases(self) -> List[DataDrivenTestCase]:
+    @classmethod
```

Comment:
Does it actually matter whether this is a class method or an instance method? (I don't know -- can you tell me?)